crates/common/tedge_config/src/tedge_config_cli/tedge_config.rs

```diff
@@ -513,6 +517,14 @@ define_tedge_config! {
             device_profile: bool,
         },
 
+        mapper: {
+            mqtt: {
+                /// The maximum message payload size that can be mapped to the cloud
+                #[tedge_config(example = "8192", default(variable = "C8Y_MQTT_PAYLOAD_LIMIT"))]
+                max_payload_size: u32,
```

Comment:
The maximum size of `u32` is much bigger than 256MiB. (`4294967295` > `268435455`). I think we need protection for such big input from user.

Reply:
Resolved by f32c7099318420e693a89e9afe2823e9cc71b5a7